lib/client.ml

```diff
@@ -1,40 +1,44 @@
 open Lwt_unix
-open Util
 
-type 'a t =
-  { address: string
-  ; port: int
-  ; socket: file_descr
-  ; state: 'a ref
-  ; recv_mutex: Lwt_mutex.t
-  ; state_mutex: Lwt_mutex.t
-  }
+type 'a t = {
+  address : string;
+  port : int;
+  socket : file_descr;
+  state : 'a ref;
+  recv_mutex : Lwt_mutex.t;
+  state_mutex : Lwt_mutex.t;
+}
 
-let peer_from (client: 'a t) =
-  Peer.{ address = client.address; port = client.port}
+let peer_from (client : 'a t) =
+  Peer.{ address = client.address; port = client.port }
 
 let send_to client payload peer =
   let len = Bytes.length payload in
   let addr = Peer.to_sockaddr peer in
   let%lwt _ = sendto !client.socket payload 0 len [] addr in
   Lwt.return ()
 
+let send_to_peers client payload peers =
```

Comment:
In order to prepare the gossip, I just created this function, to send the same message to several peers.

Reply:
Ok for the new naming, clearly better!
I didn't know `eta conversion` was present in OCaml! That's great!